deploy/remote.sh

```diff
@@ -3,42 +3,43 @@
 set -e
 
 BRANCH=$1
+backend_dir=foodsaving-backend
```

Comment:
I didn't see more places right now, and generally I'm overwhelmed with too generic things ;)